src/modules/navigator/navigation.h

```diff
@@ -49,7 +49,7 @@
 #elif defined(__PX4_POSIX)
 #  define NUM_MISSIONS_SUPPORTED (UINT16_MAX-1) // This is allocated as needed.
 #else
-#  define NUM_MISSIONS_SUPPORTED 2000 // This allocates a file of around 181 kB on the SD card.
+#  define NUM_MISSIONS_SUPPORTED 500
```

Comment:
This seems ok although note that a MAVSDK mission with 100 waypoints easily requires 500 mavlink mission items as it expands more complex mission items into several mavlink mission items.

Reply:
Ok, let me know if it becomes an issue and could find a way to make it more configurable or dynamic (at init).